src/sagemaker/mxnet/model.py

```diff
@@ -175,22 +157,41 @@ def prepare_container_def(self, instance_type, accelerator_type=None):
             deploy_image, self.repacked_model_data or self.model_data, deploy_env
         )
 
-    def serving_image_uri(self, region_name, instance_type):
+    def serving_image_uri(self, region_name, instance_type, accelerator_type=None):
         """Create a URI for the serving image.
 
         Args:
             region_name (str): AWS region where the image is uploaded.
             instance_type (str): SageMaker instance type. Used to determine device type
                 (cpu/gpu/family-specific optimized).
+            accelerator_type (str): The Elastic Inference accelerator type to
+                deploy to the instance for loading and making inferences to the
+                model (default: None). For example, 'ml.eia1.medium'.
 
         Returns:
             str: The appropriate image URI based on the given parameters.
 
         """
-        return fw_utils.create_image_uri(
+        framework_name = self.__framework_name__
+        if self._is_mms_version():
+            framework_name += "-serving"
```

Comment:
String join is more efficient than string concatenation. For more details, see:
- https://stackoverflow.com/questions/3055477/how-slow-is-pythons-string-concatenation-vs-str-join
- https://waymoot.org/home/python_string/

```suggestion
            framework_name = "-".join([framework_name, "serving"])
```

Reply:
talked offline - we'll go with format strings as a compromise between performance and readability, since the performance gain is negligible in this case

an aside - fwiw, a tuple would be more efficient than a list because it's immutable